main.go

```diff
@@ -30,11 +33,26 @@ func main() {
 	if err := env.Parse(&cfg); err != nil {
 		log.Fatalf("%+v\n", err)
 	}
+	cronjob := cron.New()
+	ctx := context.Background()
+	redis := database.InitRedis(cfg.VMStateURL)
+
+	s := &scheduler.Scheduler{
+		Redis: redis, 
+		Context: &ctx,
+	}
+	//Now inject the scheduler into the routes that need it! 
+
+	err := cronjob.AddFunc("0 */6 * * *", func() { s.FindAndDestroyUnsused() })
```

Comment:
Do we really need a library to do this ? 

Sounds like it could be a loop with a sleep in a separate thread (using the `go` keyword)